### PR TITLE
feat(workflow): Add button to duplicate alerts from the rule details page

### PR DESCRIPTION
**Pull request by snigdhas**
Add a button to duplicate a metric or issue alert from the rule details page.

This replaces the 'Give Feedback' button for Issue Alerts. 

Fixes WOR-1940

### CODE REVIEW

File: static/app/views/alerts/rules/issue/details/ruleDetails.tsx [hunk 1]
```diff
@@ -245,11 +255,8 @@ class AlertRuleDetails extends AsyncComponent<Props, State> {
           </Layout.HeaderContent>
           <Layout.HeaderActions>
             <ButtonBar gap={1}>
-              <Button
-                title={t('Send us feedback via email')}
-                href="mailto:alerting-feedback@sentry.io?subject=Issue Alert Details Feedback"
-              >
-                {t('Give Feedback')}
+              <Button title={t('Duplicate')} to={duplicateLink}>
+                {t('Duplicate')}
```
Comment (scttcper) [inline static/app/views/alerts/rules/issue/details/ruleDetails.tsx:259]: No icon in the design?

File: static/app/views/alerts/rules/metric/details/header.tsx [hunk 2]
```diff
@@ -40,6 +50,9 @@ function DetailsHeader({hasMetricRuleDetailsError, rule, params, project}: Props
           ]}
         />
         <Controls>
+          <Button title={t('Duplicate')} to={duplicateLink}>
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/details/header.tsx:53]: same here
```suggestion
          <Button to={duplicateLink}>
```

File: static/app/views/alerts/rules/issue/details/ruleDetails.tsx (hunk 1 above)
Comment (snigdhas) [reply static/app/views/alerts/rules/issue/details/ruleDetails.tsx:259]: Missed that, thanks

File: static/app/views/alerts/rules/issue/details/ruleDetails.tsx [hunk 3]
```diff
@@ -245,11 +255,8 @@ class AlertRuleDetails extends AsyncComponent<Props, State> {
           </Layout.HeaderContent>
           <Layout.HeaderActions>
             <ButtonBar gap={1}>
-              <Button
-                title={t('Send us feedback via email')}
-                href="mailto:alerting-feedback@sentry.io?subject=Issue Alert Details Feedback"
-              >
-                {t('Give Feedback')}
+              <Button title={t('Duplicate')} to={duplicateLink}>
```
[the remark this answers is not in the document]
Comment (snigdhas) [reply static/app/views/alerts/rules/issue/details/ruleDetails.tsx:258]: TIL

**Review by scttcper — APPROVED** [review]

**Pull request merged**